src/components/Home/index.tsx

```diff
@@ -1,15 +1,66 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { CameraIcon, CodeBracketIcon, CpuChipIcon, DocumentTextIcon, ShieldCheckIcon, UsersIcon } from '@heroicons/react/24/solid';
 import JetIcon from '@site/src/icons/jet.icon.svg';
 import { BlogProps } from '@site/src/types';
 import BetaflightLayout from '../Layout';
 import HomepageFeature from '../HomepageFeature';
 import AboutCard from '../AboutCard';
+import FancyAboutCard from '../FancyAboutCard';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faPatreon, faPaypal } from '@fortawesome/free-brands-svg-icons';
 import Button from '../Button';
 import TeamFeature from '../Team';
 
+function Logo() {
+  const [theme, setTheme] = useState('light');
+
+  useEffect(() => {
+    const html = document.querySelector('html');
+    const currentTheme = html.getAttribute('data-theme');
+    setTheme(currentTheme);
+    const observer = new MutationObserver((mutationsList) => {
+      for (const mutation of mutationsList) {
+        if (mutation.type === 'attributes' && mutation.attributeName === 'data-theme') {
+          setTheme(mutation.target.getAttribute('data-theme'));
+        }
+      }
+    });
+    observer.observe(html, { attributes: true });
+    return () => observer.disconnect();
+  }, []);
```

Comment:
I don't think this is the correct approach, how to set the dark theme, the logic should be at least extracted. Let's use context for storing the dark theme, it should be also persisted in localstorage. 
Some refs: e.g. 
https://javascript.plainenglish.io/how-to-implement-dark-light-themes-in-a-next-js-app-using-context-hook-tailwindcss-336558dd4579